### ISSUE TITLE
Trojan flag

### ISSUE THREAD

**Issue by TymislawMiau** [cregx/wim-backup#7]
Windows defender flags files as malicious ( as a trojan)
pls fix

**Comment by cregx** [comment]
Hi, to better understand the issue, I have a few questions:

1. **Which files** are being flagged by Windows Defender as potentially malicious or as a trojan?
2. A **screenshot** or the full text of the warning would be very helpful.
3. Have you already scanned the files with any other antivirus programs or online scanning like **virustotal.com** services? If so, did you receive similar warnings?

**Comment by cregx** [comment]
Hi, I could not detect any anomalies or problems with the released version. I have analysed version 1.14 as an example.

- Code-signed release version.
- File hash value on Virustotal.com for wimbckup.exe (0/73): [68e1f7b88af6332f8269de9f70e359cff7b6d295725198e87a9cb8f7aca8350a](https://www.virustotal.com/gui/file/68e1f7b88af6332f8269de9f70e359cff7b6d295725198e87a9cb8f7aca8350a)

![ResultWindowsDefenderScanWimBackup114](https://github.com/user-attachments/assets/e8657915-f483-4d75-bc69-a8d5bef9e602)

Best regards

**Comment by TymislawMiau** [comment]
Hello I just saw that it was flagged as a Trojan by windows defender maybe was a false flag but I’m glad that you looked into itBest regardsSimon aka TymislawMiauWiadomość napisana przez cregx ***@***.***> w dniu 11 lis 2024, o godz. 16:29:﻿
Hi, I could not detect any anomalies or problems with the released version. I have analysed version 1.14 as an example.

Code-signed release version.
File hash value on Virustotal.com for wimbckup.exe (0/73): 68e1f7b88af6332f8269de9f70e359cff7b6d295725198e87a9cb8f7aca8350a

ResultWindowsDefenderScanWimBackup114.png (view on web)
Best regards
